### PR TITLE
fix: cardInfoUtil startTime convert

### PR DESCRIPTION
**Pull request by MatoPlus**
### Implementation description

- we want explicit date conversion when converting API results into Date objects

### Steps to test

1. Program Details Page actually loads properly without client error

### CODE REVIEW

File: utils/cardInfoUtil.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export class CardInfoUtil {
             startDate: new Date(result.startDate),
             endDate: new Date(result.endDate),
             weekday: result.weekday,
-            startTimeMinutes: totalMinutes(result.startDate),
+            startTimeMinutes: result.startDate ? totalMinutes(new Date(result.startDate)) : 0,
```
Comment (feng-eric) [inline utils/cardInfoUtil.ts:51]: Pretty minor but on line 48, you already have `new Date(result.startDate)`. Up to you if you want to reduce duplicate code.

Comment (feng-eric) [reply utils/cardInfoUtil.ts:51]: Also, in the cases when the ternary expression is false, what would `new Date(result.startDate)` return on line 48? Should there be a ternary expression there too then?

**Review by feng-eric — DISMISSED** [review]

File: utils/cardInfoUtil.ts (hunk 1 above)
Comment (MatoPlus) [reply utils/cardInfoUtil.ts:51]: true, I'll do new Date() in for false I guess.

Comment (MatoPlus) [reply utils/cardInfoUtil.ts:51]: I think I'll just leave the one dup line in for now. I think it should be fine with how we're doing ternaries.

**Review by feng-eric — APPROVED** [review]

**Pull request merged**